src/renderer/features/accounts-structure/components/AccountStructureNode.tsx

```diff
@@ -63,7 +65,7 @@ export const AccountStructureNode = memo(({ data, id }: AccountStructureNodeProp
             <div className="px-4 py-2 text-sm text-text-secondary">
               <Address
                 address={toAddress(data.node.account.accountId, { prefix: chain?.addressPrefix })}
-                title={data.node.account.name}
+                title={accountIdentity?.name ?? data.node.account.name}
```

Comment:
i think it should be reversed - user defined name OR network identity as fallback